integrationtests/Paket.IntegrationTests/RestoreSpecs.fs

```diff
@@ -42,4 +42,55 @@ let ``#2496 Paket fails on projects that target multiple frameworks``() =
    
     match (result.OK, hasErrorOut) with 
     | (true, false) -> printfn "all good!"
+    | _ -> failwithf "dotnet restore exitCode %d > 0? or hasErrorOut %b" exitCode hasErrorOut
+    
+[<Test>]
+let ``#xxx dotnet restore writes paket references file to correct obj dir``() = 
+    let project = "ObjDir"
+    let scenario = "i00xxxx-obj-dir"
+    let tmpPaketFolder = (scenarioTempPath scenario) @@ "../.paket"
+    let targetsFile = FullName(__SOURCE_DIRECTORY__ + "../../../src/Paket/embedded/Paket.Restore.targets")
+    let paketExe = FullName(__SOURCE_DIRECTORY__ + "../../../.paket/paket.exe")
```

Comment:
This is wrong (and is also wrong in the test above): it copies the checked-in version of paket, not the previously build one.

Tomorrow ...